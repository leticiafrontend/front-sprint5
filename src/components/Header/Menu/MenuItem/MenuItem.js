import React from 'react'
import { Item, Link } from './style'

export const MenuItem = ({ link, label }) => (
  <Item>
    <Link href={link}>{label}</Link>
  </Item>
)
