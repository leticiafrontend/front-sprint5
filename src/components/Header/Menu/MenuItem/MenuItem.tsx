import React from 'react';
import { Item, Link } from './style';

interface PropsMenuItem {
  link: string;
  label: string;
}

export const MenuItem: React.FC<PropsMenuItem> = ({ link, label }) => (
  <Item>
    <Link href={link}>{label}</Link>
  </Item>
);
