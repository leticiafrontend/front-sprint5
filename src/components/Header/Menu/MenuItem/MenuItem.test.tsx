import React from 'react'
import { render } from '@testing-library/react'
import { MenuItem } from './MenuItem'

describe('Componente de item do menu', () => {
  it('O snapshot do componente deve permanecer sempre o mesmo', () => {
    const { container } = render(<MenuItem link='/home' label='Home' />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
