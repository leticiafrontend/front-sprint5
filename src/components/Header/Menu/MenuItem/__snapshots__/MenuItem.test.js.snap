// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente de item do menu O snapshot do componente deve permanecer sempre o mesmo 1`] = `
<li
  class="sc-bdnxRM"
>
  <a
    class="sc-gtsrHT kvHodI"
    href="/home"
  >
    Home
  </a>
</li>
`;
